modules/troubleshoot/pages/debug-bundle/configure/kubernetes.adoc

```diff
@@ -0,0 +1,7 @@
+= Configure Debug Bundle Management in Kubernetes
+:env-kubernetes: ''
```

Comment:
should this be empty?

Reply:
In Asciidoc, an empty string means the attribute is truthy

Reply:
I would add true just to be explicit, since you do this in the other doc.